commands/config_test.go

```diff
@@ -1,104 +0,0 @@
-package commands
```

Comment:
It is unfortunate to have to remove these tests (they were very hardcoded to the old `provider` system but I am replacing them with BATS-based integration tests (if you look closely, you'll notice these are basically integration tests anyway).